### PR TITLE
Changed serialization for redis from json into MessagePack

### PR DESCRIPTION
**Pull request by polonzer**
https://lykkex.atlassian.net/browse/LWDEV-4765
+ added cache InstanceName to config.

### CODE REVIEW

File: src/Lykke.Job.OperationsCache/Settings/JobSettings/OperationsCacheSettings.cs [hunk 1]
```diff
@@ -1,10 +1,13 @@
 ﻿using System;
 using System.Collections.Generic;
+using Lykke.SettingsReader.Attributes;
 
 namespace Lykke.Job.OperationsCache.Settings.JobSettings
 {
     public class OperationsCacheSettings
     {
+        [Optional]
```
Comment (NovichikhinAlexey) [inline src/Lykke.Job.OperationsCache/Settings/JobSettings/OperationsCacheSettings.cs]: why settings optional?

Comment (NovichikhinAlexey) [reply src/Lykke.Job.OperationsCache/Settings/JobSettings/OperationsCacheSettings.cs]: let's make all settings property as Required

Comment (NovichikhinAlexey) [reply src/Lykke.Job.OperationsCache/Settings/JobSettings/OperationsCacheSettings.cs]: becouse after admins and another piople can not find this setting or the one who writes to the index HistoryCacheInstance

Comment (polonzer) [reply src/Lykke.Job.OperationsCache/Settings/JobSettings/OperationsCacheSettings.cs]: ok

**Review by NovichikhinAlexey — APPROVED** [review]

**Pull request merged**